src/pydata_sphinx_theme/theme/pydata_sphinx_theme/sections/sidebar-primary.html

```diff
@@ -6,7 +6,13 @@
     {% if theme_navbar_center %}
       <div class="sidebar-header-items__center">
         {% for navbar_item in theme_navbar_center %}
-          <div class="navbar-item">{% include navbar_item %}</div>
+          {#
+            we are in the sidebar for narrow screnns for example, we do not want
+            an dropdown, so set the max drop value to a large number Here 999.
+          #}
+          {% with theme_header_links_before_dropdown=999 %}
```

Comment:
Ah I wish I had thought of this hack myself, love it!

Reply:
Yeah – it took me a while to find it. But honestly I think we should not use include but named macros to track param, though it's going to be hard to refactor to do so.